api/src/main/java/io/strimzi/api/kafka/model/JmxTransSpec.java

```diff
@@ -46,7 +46,8 @@
 
     @JsonProperty(value = "outputDefinitions", required = true)
     @JsonInclude(JsonInclude.Include.NON_DEFAULT)
-    @Description("Defines the output hosts that will be referenced later on")
+    @Description("Defines the output hosts that will be referenced later on. " +
+            "For more information, see `proc-jmxtrans-deployment.adoc`")
```

Comment:
not sure that we can reference a doc page in this way. @PaulRMellor @laidan6000 ?

Reply:
No, I'd rather we didn't include a reference in this format.

Reply:
@julianGoh17 maybe a reference to the official jmxtrans page?

Reply:
How would I reference the internal documentation? The reason I am hesitant to reference the JmxTrans Github is because not every single query configuration can currently be used. I have included the most useful and important ones in this PR but there is room to add the rest in a future PR.

Reply:
@julianGoh17 - I'm not sure that you can reference the internal (Strimzi) documentation from within a Java description class. Using an Asciidoc cross-reference (`x-ref`) will not work as far as I know. @scholzj - Do you have any suggestions?